pkgs/tools/misc/zalgo/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zalgo";
+  version = "6aa1f66";
```

From the [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming): `If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".`

```suggestion
  pname = "zalgo-unstable";
  version = "2020-08-26";
```